client/site-profiler/components/results-header/index.tsx

```diff
@@ -10,14 +10,17 @@ type Props = {
 	domain: string;
 	performanceCategory: PerformanceCategories;
 	isWpCom: boolean;
+	isWordPress: boolean;
 	onGetReport: () => void;
 };
 
-function getIcon( isWpCom?: boolean ) {
-	if ( isWpCom ) {
-		return <img src={ wpComSiteIcon } alt={ translate( 'WordPress.com site' ) } />;
+function getIcon( isWordPress?: boolean ) {
+	if ( isWordPress ) {
+		return <img src={ wpComSiteIcon } alt={ translate( 'This site is built with WordPress' ) } />;
```

Comment:
I think this text should also be shown to the user, instead of being only an `alt` text.

Reply:
That's a good idea! I updated the component to output more variations on the message, if the site is built with WordPress. Here is how it looks together with the icon:

| Site | Message |
| --- | --- |
| Non-WP site | ![CleanShot 2024-06-07 at 14 56 36@2x](https://github.com/Automattic/wp-calypso/assets/11555574/6cf24ca0-b988-4921-b58c-850d8e23ee64) |
| WP site not on WP.com | ![CleanShot 2024-06-07 at 14 50 22@2x](https://github.com/Automattic/wp-calypso/assets/11555574/fc09c14a-62a5-40fc-a735-e274e2c44382)  |
| WP site on WP.com | ![CleanShot 2024-06-07 at 14 52 45@2x](https://github.com/Automattic/wp-calypso/assets/11555574/6cf97325-43f0-4172-840c-0d2830265f92) |

Reply:
As a last suggestion I would just keep the message for WP.com as: `This site is hosted on WordPress.com` because this infers its a WordPress site.

Reply:
Just wondering if this should be an `alt` text at all - it strikes me as unnecessary if the text is already visible, as it’ll just repeat on screen readers twice?

Reply:
Thanks @Aurorum, that's a good point. I updated the alt tag in 84babd7745